Lib/test/test__xxsubinterpreters.py

```diff
@@ -105,6 +105,7 @@ def test_default_shareables(self):
                 True,
                 False,
                 100.0,
+                (),
```

Comment:
For this test consider making the tuple an interesting one:

```suggestion
                (1, ('spam', 'eggs')),
```

We cover the various boring cases in the new `test_tuple()` below.